### DISCUSSION TITLE
Stuck with Fos JS Routing & Asset Mapper – Help Me Tackle This JSON Modules Error!

### DISCUSSION THREAD

**Discussion by ahmed-bhs** [symfony/symfony#59131] [Q&A]
Hey folks, I'm trying to get Fos JS Routing working with Asset Mapper, but I keep running into this error:

Uncaught Error: json-modules feature must be enabled via <script type="esms-options">{ "polyfillEnable": ["json-modules"] }</script>

Here's what I've tried so far:

```
import routes from './js/fos_js_routes.json';
import Routing from "fos-router";

Routing.setRoutingData(routes);
window.Routing = window.Routing = Routing;
```

Anyone have a clue on how to fix this or make it work properly? Would really appreciate the help!

**Comment by MatTheCat** [comment]
Hello, did you try what the error message suggests..? Seems you need to activate the polyfill’s JSON modules support: https://github.com/guybedford/es-module-shims?tab=readme-ov-file#json-modules
